### PR TITLE
Added default value and predefined values to argument description

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29972

## Description
Added optional values to argument description (if exists)
Added default value to argument description (if exists)

### CODE REVIEW

File: demisto_sdk/commands/generate_docs/generate_integration_doc.py [hunk 1]
```diff
@@ -248,8 +248,13 @@ def generate_single_command_section(cmd: dict, example_dict: dict, command_permi
             if not arg.get('description'):
                 errors.append(
                     'Error! You are missing description in input {} of command {}'.format(arg['name'], cmd['name']))
+            argument_description = f'{arg.get("description")} The optional values are {arg.get("predefined")}.' \
```
Comment (Itay4) [inline demisto_sdk/commands/generate_docs/generate_integration_doc.py:251]: please work with @kirbles19 on the text

Comment (kirbles19) [reply demisto_sdk/commands/generate_docs/generate_integration_doc.py:251]: Possible values are: {arg.get("predefined")}. Default is {arg.get("defaultValue")}.

**Review by kirbles19 — APPROVED** [review]

**Comment by kirbles19** [comment]
@Shellyber Done.

File: demisto_sdk/commands/generate_docs/generate_integration_doc.py (hunk 1 above)
Comment (Itay4) [reply demisto_sdk/commands/generate_docs/generate_integration_doc.py:251]: @Shellyber note the value for the possible values need to be comma separated without the squared brackets, so you will need to use `join`

Comment (Shellyber) [reply demisto_sdk/commands/generate_docs/generate_integration_doc.py]: @Itay4 Done. Please approve also

File: demisto_sdk/commands/generate_docs/generate_integration_doc.py [hunk 2]
```diff
@@ -248,8 +248,13 @@ def generate_single_command_section(cmd: dict, example_dict: dict, command_permi
             if not arg.get('description'):
                 errors.append(
                     'Error! You are missing description in input {} of command {}'.format(arg['name'], cmd['name']))
+            argument_description = f'{arg.get("description")} Possible values are: {", ".join(arg.get("predefined"))}.'\
```
Comment (Itay4) [inline demisto_sdk/commands/generate_docs/generate_integration_doc.py:251]: please handle the case the description does not end with a dot by adding one

File: demisto_sdk/commands/generate_docs/generate_integration_doc.py [hunk 3]
```diff
@@ -248,8 +248,16 @@ def generate_single_command_section(cmd: dict, example_dict: dict, command_permi
             if not arg.get('description'):
                 errors.append(
                     'Error! You are missing description in input {} of command {}'.format(arg['name'], cmd['name']))
+            if not arg.get('description').endswith('.'):
```
Comment (Itay4) [inline demisto_sdk/commands/generate_docs/generate_integration_doc.py:251]: no need to read from the dictionary 3 times and assign to it
just init a variable with it and work with that instead

**Review by Itay4 — APPROVED** [review]

**Pull request merged**